### DISCUSSION TITLE
People Search Profile Update Options

### DISCUSSION THREAD

**Discussion by AravindSarathy** [microsoft-search/pnp-modern-search#3914] [Q&A]
Hi Everyone,

I would like to know if editing user profile is possible when searching users using this modern search webpart.
As Microsoft has announced that Delve will be retired soon, will the "Update Profile" option in the profile cards allow us to update the details within itself?

I would appreciate if anyone can let know whether this change will be accommodated in this webpart in future

Note: Currently we see that "Update Profile" is pointing tot he Delve site.

**Comment by kasperbolarsen** [comment]
Wictor have given some guidance here: https://www.linkedin.com/posts/wictorwilen_alternatives-to-delve-in-microsoft-365-activity-7186614139463581696-8JOL/

and NO, we will not provide any option to update user profile data as that is most certainly on Microsofts part of the playing field

**Comment by wobba** [comment]
Update of profile properties will continue to be supported. Meaning any Delve URL will redirect to an experience allowing edit.
